skyvern/services/script_service.py

```diff
@@ -2187,6 +2187,7 @@ async def prompt(
     label: str | None = None,
     parameters: list[str] | None = None,
     model: dict[str, Any] | None = None,
+    llm_key: str | None = None,
```

Comment:
refer to this part to see how model override works: https://github.com/Skyvern-AI/skyvern-cloud/blob/2aeaf7c17e62031615de515fc8c6ff094d3905b5/skyvern/forge/sdk/workflow/models/block.py#L1890-L1892

`llm_key` should be a deprecated value. we should just have the `model: dict | None` parameter here